addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -545,44 +457,18 @@ def test_channel_private_unfollow(self):
         # sudo: discuss.channel - reading members of non-accessible channel for testing purposes
         self.assertEqual(len(chat_user_current.sudo().channel_member_ids), 0)
 
-    def test_channel_unfollow_should_not_post_message_if_the_partner_has_been_removed(self):
-        '''
-        When a partner leaves a channel, the system will help post a message under
-        that partner's name in the channel to notify others if `email_sent` is set `False`.
-        The message should only be posted when the partner is still a member of the channel
-        before method `_action_unfollow()` is called.
-        If the partner has been removed earlier, no more messages will be posted
-        even if `_action_unfollow()` is called again.
-        '''
+    def test_channel_unfollow_should_not_post_message(self):
```

Comment:
This test should not remove the assertion, rather use a channel with a type allowing the notif (group I guess)